AUTHORS

```diff
@@ -1,4 +1,4 @@
 - Juergen Boehringer (see www.boehri.de)
 - Reinhard X. Fuerst (see feinstaub.rexfue.de)
 - Thomas Waldmann <twaldmann@thinkmo.de>
-
+- and others
```

Comment:
:-) feel free to add your name.

Reply:
nein auf keinen Fall, aber nun haben hier ja schon ein paar andere etwas beigetragen, daher wollte ich das kenntlich machen 😄